examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<p>Please provide more details. E.g. what item you select etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>